### PR TITLE
Implements a landing page intro

### PR DESCRIPTION
**Pull request by JonPhoenix**

### CODE REVIEW

File: client/src/components/LandingIntro/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export function LandingIntro() {
+  return (
+    <>
+      <div id="intro" className="intro-container">
+        <h1 className="intro-title">Yakan-do-it!</h1>
+        <h2 className="intro-text">
+          The <span className="intro-title">5 Sherpas</span> give you
+          personalized suggestions to achieve a healthier and more balanced life
+          through activities called weekly challenges.
+          <br></br>
```
Comment (zapponejosh) [inline client/src/components/LandingIntro/index.js:13]: @JonPhoenix  can you change these to `<br />`

File: client/src/pages/LandingPage/index.js [hunk 2]
```diff
@@ -25,6 +26,19 @@ function LandingPage() {
         <div className="register-btn">
           <RegisterButton />
         </div>
+        <div id="know">
+          <a id="link" href="#intro">
+            Know <br></br>More
```
Comment (zapponejosh) [inline client/src/pages/LandingPage/index.js:31]: this one too

File: client/src/components/LandingIntro/index.js (hunk 1 above)
Comment (JonPhoenix) [reply client/src/components/LandingIntro/index.js]: Done!

File: client/src/pages/LandingPage/index.js (hunk 2 above)
Comment (JonPhoenix) [reply client/src/pages/LandingPage/index.js]: Done!

**Review by zapponejosh — APPROVED** [review]

**Pull request merged**